installer.nsh

```diff
@@ -18,9 +18,16 @@
     ; stop the agent
     nsExec::ExecToStack /OEM 'powershell "& " "$\'"$path_i\remoteit.exe$\'" -j agent stop'
 
-    ; move the config file and connections to backup location (protect against 2.9.2 uninstall bug)
-    Rename "$APPDATA\remoteit\config.json" "${REMOTEIT_BACKUP}\config.json"
-    Rename "$PROFILE\AppData\Local\remoteit\connections" "${REMOTEIT_BACKUP}\connections"
+    ; move the config file and connections to backup location ONLY MOVES IF EMPTY (protect against 2.9.2 uninstall bug)
+    ; Rename "$APPDATA\remoteit\config.json" "${REMOTEIT_BACKUP}\config.json"
+    ; Rename "$PROFILE\AppData\Local\remoteit\connections" "${REMOTEIT_BACKUP}\connections"
+
+    ; copy the config file and connections to backup location
+    CopyFiles /SILENT "$APPDATA\remoteit\config.json" "${REMOTEIT_BACKUP}\"
```

Comment:
You have to get the config.json out of /etc/remoteit OR 2.9.0 installer will unregister the devices

Reply:
No use move and NOT copy